### PR TITLE
[WIP] start constants.py

### PR DESCRIPTION
**Pull request by williambaibai**
Just want to get opinions on whether this is a good way to do constants, before I proceed with the rest of the hard-coded data. Thanks!

### CODE REVIEW

**Comment by williambaibai** [comment]
@kq-li @henrytso @DanielNosrati finished gyms constants

File: constants.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+        name='Teagle Down',
+        description='description',
+        popular=[
+            [0, 0, 0, 0, 0, 0, 0, 14, 27, 41, 53, 60, 58, 50, 44, 45, 56, 69, 74, 64, 43, 22, 80, 0],
+            [0, 0, 0, 0, 0, 0, 0, 16, 26, 35, 43, 53, 63, 67, 61, 53, 50, 52, 51, 44, 30, 16, 6, 0],
+            [0, 0, 0, 0, 0, 0, 0, 21, 46, 38, 40, 58, 62, 48, 34, 35, 51, 68, 75, 65, 47, 26, 11, 0],
+            [0, 0, 0, 0, 0, 0, 0, 16, 26, 37, 46, 52, 53, 52, 51, 53, 53, 47, 45, 58, 59, 32, 7, 0],
+            [0, 0, 0, 0, 0, 0, 0, 12, 26, 32, 38, 48, 56, 54, 50, 52, 53, 44, 26, 11, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 9, 17, 27, 36, 41, 36, 24, 0, 0, 0, 0, 0],
+            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 3, 13, 21, 24, 34, 36, 14 ,0 ,0 ,0 ,0 ,0]
```
Comment (kq-li) [inline constants.py]: `..., 14, 0, 0, 0, 0, 0]`

File: constants.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+            DayTimeRangeType(day=0, start_time=dt.time(7), end_time=dt.time(22, 45)),
+            DayTimeRangeType(day=1, start_time=dt.time(7), end_time=dt.time(22, 45)),
+            DayTimeRangeType(day=2, start_time=dt.time(7), end_time=dt.time(22, 45)),
+            DayTimeRangeType(day=3, start_time=dt.time(7), end_time=dt.time(22, 45)),
+            DayTimeRangeType(day=4, start_time=dt.time(7), end_time=dt.time(20)),
+            DayTimeRangeType(day=5, start_time=dt.time(12), end_time=dt.time(17, 45)),
+            DayTimeRangeType(day=6, start_time=dt.time(12), end_time=dt.time(17, 45))
+        ]
+    ),
+    GymType(
```
Comment (kq-li) [inline constants.py]: honestly just leave bartels off this list

File: constants.py [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    )
+]
+
+GYMS = {
+    GYMS_LIST[0].id: GYMS_LIST[0],
+    GYMS_LIST[1].id: GYMS_LIST[1],
+    GYMS_LIST[2].id: GYMS_LIST[2],
+    GYMS_LIST[3].id: GYMS_LIST[3],
+    GYMS_LIST[4].id: GYMS_LIST[4]
+}
```
Comment (kq-li) [inline constants.py]: python dict comprehensions might come in handy here

**Pull request merged**